.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,9 +12,8 @@
 > /kind flake
 > /kind code-refactoring
 >
-> Documentation changes: Please include [skip ci] in your commit message as well
+> Documentation changes: CI would automatically not run tests based on which files have been edited.
```

Comment:
This is a little confusing to me. Can we just make it say that CI will automatically skip test runs if it is a doc related change?

Reply:
Sure